notebooks/240-dolly-2-instruction-following/240-dolly-2-instruction-following.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    %pip install -q "git+https://github.com/openvinotoolkit/nncf.git@release_v260"
</pre><p>%pip install "nncf&gt;=2.6.0"</p><p>%pip install "openvino-dev&gt;=2023.1"</p><p>(openvino package install included in openvino-dev, no need to install explicitly, however, optimum dependency on openvino-dev will be removed in https://github.com/huggingface/optimum-intel/pull/397 and only openvino will be necessary to install)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Added only %pip install "openvino-dev>=2023.1", since NNCF 2.6 is now listed in requirements.txt